<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNew</class>
 <widget class="QDialog" name="DialogNew">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Invoice</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/res/document-new.png</normaloff>:/res/res/document-new.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Cost:</string>
       </property>
       <property name="buddy">
        <cstring>doubleSpinBox_cost</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QComboBox" name="comboBox_status">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Description:</string>
       </property>
       <property name="buddy">
        <cstring>plainTextEdit_description</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QLineEdit" name="lineEdit_name">
       <property name="placeholderText">
        <string>Type a name...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_10">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Phone:</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_phone</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Device:</string>
       </property>
       <property name="buddy">
        <cstring>comboBox_device</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="5">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_cost">
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;Tahoma&quot;;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::PlusMinus</enum>
       </property>
       <property name="suffix">
        <string> LD</string>
       </property>
       <property name="maximum">
        <double>99999999.989999994635582</double>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QDateEdit" name="dateEdit_delivered">
       <property name="displayFormat">
        <string>yyyy/MM/dd</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_name</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="5">
      <widget class="QPlainTextEdit" name="plainTextEdit_description">
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Recived:</string>
       </property>
       <property name="buddy">
        <cstring>dateEdit_recived</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_11">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Status:</string>
       </property>
       <property name="buddy">
        <cstring>comboBox_status</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_delivered">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Delivered:</string>
       </property>
       <property name="buddy">
        <cstring>dateEdit_delivered</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QDateEdit" name="dateEdit_recived">
       <property name="displayFormat">
        <string>yyyy/MM/dd</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBox_device">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Model:</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_model</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="label_9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Serial:</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_serial</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QLineEdit" name="lineEdit_serial">
       <property name="placeholderText">
        <string>Serial number</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Note:</string>
       </property>
       <property name="buddy">
        <cstring>plainTextEdit_note</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="5">
      <widget class="QPlainTextEdit" name="plainTextEdit_note">
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_phone">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Phone number...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="lineEdit_model"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_delete">
        <property name="text">
         <string>Delete</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/res/edit-delete.png</normaloff>:/res/res/edit-delete.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_add">
        <property name="text">
         <string>Add New</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/res/list-add.png</normaloff>:/res/res/list-add.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_ok">
        <property name="text">
         <string>Save</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/res/dialog-ok-apply.png</normaloff>:/res/res/dialog-ok-apply.png</iconset>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_cancel">
        <property name="text">
         <string>Close</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/res/dialog-close.png</normaloff>:/res/res/dialog-close.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_name</tabstop>
  <tabstop>comboBox_status</tabstop>
  <tabstop>lineEdit_phone</tabstop>
  <tabstop>dateEdit_recived</tabstop>
  <tabstop>comboBox_device</tabstop>
  <tabstop>lineEdit_model</tabstop>
  <tabstop>lineEdit_serial</tabstop>
  <tabstop>plainTextEdit_description</tabstop>
  <tabstop>doubleSpinBox_cost</tabstop>
  <tabstop>plainTextEdit_note</tabstop>
  <tabstop>pushButton_add</tabstop>
  <tabstop>pushButton_ok</tabstop>
  <tabstop>pushButton_cancel</tabstop>
  <tabstop>pushButton_delete</tabstop>
  <tabstop>dateEdit_delivered</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>DialogNew</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>591</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
